test/cypress/integration/signup-and-login.spec.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-env mocha */
```

Comment:
I believe we don't need this comment in `cypress/` files. Please remove them across all cypress files.